src/components/common/validation/time-format/ValidatedTimeEntry.spec.jsx

```diff
@@ -41,7 +41,7 @@ describe('ValidatedTimeEntry', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should call onblur method', async () => {
+  it('should call setFinishTimeText method', async () => {
```

Comment:
Should the `fireEvent.blur `for this test be `fireEvent.onChange` now?

Reply:
I assumed blur would also trigger the onChange code, but looking at the test, if I comment the whole act step out, it still passes, so I'll have a look now...

Reply:
I've removed this test and added one to the EditShiftHours to cover the same functionality.

Reply:
Looks good to me